CODEOWNERS

```diff
@@ -1,5 +1,29 @@
 * @evan2645 @amartinezfayo @azdagron @APTy @marcosy
 
+##
+## SPIRE Maintainers 
+##
+
+# Evan Gilman
+# Hewlett-Packard Enterprise
+@evan2645
```

Comment:
From what I understand, this isn't proper CODEOWNER syntax. I think we need to comment out the username on each of these (the * entry at the top is assigning the default ownership)
```suggestion
# @evan2645
```